### DISCUSSION TITLE
How can I use a JavaScript variable in css

### DISCUSSION THREAD

**Discussion by uhuikim** [chakra-ui/panda#2602] [Q&A] [answered]
I'm using MUI and pandaCSS together.

I want to get the className variable defined in the mui and use it in the Panda css.
How can I do this?

```
import { alertClasses } from '@mui/material';
import { css } from '../styled-system/css'

const container = css({
  `& > ${alertClasses.root}` : {
    alignItems: 'center',
  }
});
```

**Comment by astahmer** [comment, marked as answer]
that can't be done in your app code due to the way Panda works (using static analysis, looking at each file in isolation without resolving import values); but it can be done in your `panda.config` (there is no static limitations there)

**Reply by uhuikim** [reply]
thank you~~:)
